lib/bitstyles_phoenix/component/form.ex

```diff
@@ -645,13 +677,25 @@ defmodule BitstylesPhoenix.Component.Form do
 
     ~H"""
     <.ui_unwrapped_input {@wrapper}>
-      <%= PhxForm.select(@form, @field, @options, @extra) %>
+      <%= render_input(:select, @form, @field, @extra) %>
```

Comment:
Can't we kick the `render_input` function, since it was private anyway and call `<input` directly? This also allows for better change tracking within the components.

Reply:
Done: [`aadd1c2` (#130)](https://github.com/bitcrowd/bitstyles_phoenix/pull/130/commits/aadd1c230d00c9d0528118a8947ebef64b3eeded)